### ISSUE TITLE
suggest: update webpage re Python

### ISSUE THREAD

**Issue by nbehrnd** [babluboy/bookworm#376]
The webpage https://babluboy.github.io/bookworm/#install explicitly describes Python2 as dependency (first bullet list, and the command for the CLI to let `apt` resolve all dependencies in one run).

There is no obvious file I associate with the content of the source code; for this, my criterion with [searchmonkey](https://tracker.debian.org/pkg/searchmonkey) was to find any file (no constraint regarding the file extension) with one or more occurrences of `Then execute the following commands`.  On the other hand, Python2 no longer is in active development, and https://github.com/babluboy/bookworm/tree/master/data/scripts/mobi_lib lists many scripts with the commit message of `Update mobi lib for Python 3` by [2020-04-12 Sun].

If building the executable *only* with the libraries branch of Python3 works, I suggest that the web page is going to be updated accordingly, to equally reflect this very transition.

**Comment by babluboy** [comment]
@nbehrnd You are correct, I did update the python code to be compatible with Python 3 (nothing major, just print statements etc). I will try and get eOS back on my system, compile with the latest "python3.10" and if everything is fine, then will change the web page documentation
